data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -208,6 +208,23 @@ def prepare_germline_jobs(this_batch: Batch, this_batch_run: BatchRun, this_sqr:
         # collect back BSSH run styled SampleSheet(SampleID) globally unique ID format from rgid
         sample_library_names = list(map(lambda k: k.split('.')[-1], sample_df.rgid.unique().tolist()))
 
+        # Skip samples where metadata workflow is set to manual
+        is_manual = False
+        for sample_library_name in sample_library_names:
+            library_metadata: pd.DataFrame = metadata_df.query(f"sample=='{sample_library_name}'")
+            if library_metadata["workflow"].unique().item() == "manual":
```

Comment:
Please guard DF empty check for `df.query(...)` call as it may return blank frame hence raise `KeyError`, like so:
```
if not library_metadata.empty and library_metadata["workflow"].unique().item() == "manual":
```